src/components/dev-hub/slider.js

```diff
@@ -66,10 +66,10 @@ const SliderLabelText = styled(P3)`
 `;
 
 const Slider = ({ current, currentLabel, onChange, total, totalLabel }) => {
-    const sliderPercentage = useMemo(() => Math.ceil((current / total) * 100), [
-        current,
-        total,
-    ]);
+    const sliderPercentage = useMemo(
```

Comment:
Not sure why these formatting changes made it in but having trouble removing them from the PR.